config/roles.php

```diff
@@ -64,7 +66,7 @@
     // creator roles assigned by admin
 
     [
-        'name' => Role::HUB_MANAGER,
+        'name' => Role::GAME_HASH_MANAGER,
```

Comment:
Do we need a migration for these?

Reply:
I don't think so - we're not going to be modifying any table structures. We just need to run the SQL commands in the PR description before the release.